### DISCUSSION TITLE
build error

### DISCUSSION THREAD

**Discussion by kuolung1** [Joshua-Riek/ubuntu-rockchip#396] [Q&A]
sudo ./build.sh --board=orangepi-5-plus
find: ‘build/linux-image-*.deb’: 沒有此一檔案或目錄
正複製到 'linux-rockchip'...
warning: 找不到要複製的遠端分支 linux-5.10-gen-5.1。
fatal: 遠端分支 linux-5.10-gen-5.1 在上游 origin 未發現
Error: in ./scripts/build-kernel.sh on line 15
Error: in ./build.sh on line 144

found a problem for build

**Comment by Joshua-Riek** [comment]
Thanks, fixed with https://github.com/Joshua-Riek/ubuntu-rockchip/commit/8567ab6a7edff21f59cec6fac8dfcd048123d7dd.
